app/src/main/java/com/android/shelfLife/ui/camera/CameraView.kt

```diff
@@ -259,6 +353,123 @@ fun PermissionDeniedScreen(navigationActions: NavigationActions) {
       }
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ScannedItemFoodScreen(
+    houseHoldViewModel: HouseholdViewModel,
+    foodFacts: FoodFacts,
+    foodItemViewModel: ListFoodItemsViewModel,
+    onFinish: () -> Unit // Callback to reset scanning state
+) {
+
+  var location by remember { mutableStateOf(FoodStorageLocation.PANTRY) }
+  var expireDate by remember { mutableStateOf("") }
+  var openDate by remember { mutableStateOf("") }
+  var buyDate by remember { mutableStateOf(formatTimestampToDate(Timestamp.now())) }
+
+  var locationExpanded by remember { mutableStateOf(false) }
+
+  Scaffold(
+      modifier = Modifier.fillMaxSize(),
+      topBar = {
+        TopAppBar(
+            title = { Text("Add Food Item") },
+            navigationIcon = {
+              // Back button to return to the previous screen
+              IconButton(onClick = { onFinish() }) {
+                Icon(imageVector = Icons.Default.ArrowBack, contentDescription = "Go back Icon")
+              }
+            })
+      },
+  ) { padding ->
+    Column(
+        modifier = Modifier.fillMaxSize().padding(padding).padding(16.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Top) {
+          Spacer(modifier = Modifier.height(16.dp))
+
+          ExposedDropdownMenuBox(
+              expanded = locationExpanded,
+              onExpandedChange = { locationExpanded = !locationExpanded }) {
+                OutlinedTextField(
+                    value = location.name.lowercase(),
+                    onValueChange = {},
+                    label = { Text("Location") },
+                    readOnly = true,
+                    trailingIcon = {
+                      ExposedDropdownMenuDefaults.TrailingIcon(expanded = locationExpanded)
+                    },
+                    modifier = Modifier.fillMaxWidth().menuAnchor())
+                ExposedDropdownMenu(
+                    expanded = locationExpanded, onDismissRequest = { locationExpanded = false }) {
+                      FoodStorageLocation.values().forEach { selectionOption ->
```

Comment:
Recommended to replace ` Enum.values() ` by `Enum.entries`
```suggestion
                      FoodStorageLocation.entries.forEach { selectionOption ->
```